app/src/main/java/com/carrotcreative/cream_example/app/cache/params/GithubUserLoaderParams.java

```diff
@@ -0,0 +1,24 @@
+package com.carrotcreative.cream_example.app.cache.params;
+
+import com.carrotcreative.cream.params.LoaderParams;
+
+public class GithubUserLoaderParams implements LoaderParams
+{
+    private String mUserId;
+
+    public GithubUserLoaderParams(String userId)
+    {
+        mUserId = userId;
+    }
+
+    public String getUserId()
+    {
+        return mUserId;
+    }
+
+    @Override
+    public String getIdentifier()
+    {
+        return mUserId;
```

Comment:
Even though this is a trivial example, this should probably use `getUserId()` instead, as that's how we use it most of the time in production.